app/gimlet-rot/app.toml

```diff
@@ -65,27 +65,38 @@ pins = [
 { pin = { port = 0, pin = 30}, alt = 1}
 ]
 
-[tasks.spi0_driver]
-name = "drv-lpc55-spi-server"
+[tasks.rng_driver]
+name = "drv-lpc55-rng"
 priority = 4
-max-sizes = {flash = 16384, ram = 2048}
+max-sizes = {flash = 16384, ram = 4096}
+uses = ["rng", "pmc"]
+start = true
+stacksize = 2200
+task-slots = ["syscon_driver"]
+
+[tasks.sprot]
+name = "drv-lpc55-sprot-server"
+priority = 4
+max-sizes = {flash = 32768, ram = 32768}
 uses = ["iocon", "flexcomm3", "flexcomm8"]
 features = ["spi0"]
 start = true
 interrupts = {"flexcomm8.hs_spi" = 1}
-stacksize = 1000
+stacksize = 16384
 task-slots = ["gpio_driver", "syscon_driver"]
 
-[tasks.spi0_driver.config]
+[tasks.sprot.config]
 pins = [
 # HS_SPI_MOSI = P0_26 = FUN9
-{ pin = { port = 0, pin = 26}, alt = 9},
+{ pin = { port = 0, pin = 26}, alt = 9, mode = "NoPull"},
 # HS_SPI_MISO = P1_3 = FUN6
-{ pin = { port = 1, pin = 3}, alt = 6},
+{ pin = { port = 1, pin = 3}, alt = 6, mode = "NoPull"},
 # HS_SPI_SCK = P1_2 = FUN6
-{ pin = { port = 1, pin = 2}, alt = 6},
+{ pin = { port = 1, pin = 2}, alt = 6, mode = "NoPull"},
 # HS_SSEL1 = P1_1 = FUN5
-{ pin = { port = 1, pin = 1}, alt = 5},
+{ pin = { port = 1, pin = 1}, alt = 5, mode = "NoPull"},
```

Comment:
NoPull should be the default, do you actually need to specify it here?

Reply:
Done